Dockerfile

```diff
@@ -86,6 +86,8 @@ COPY ./bin/celery_worker.sh /celery_worker.sh
 COPY ./bin/celery_beat.sh /celery_beat.sh
 COPY ./bin/celery_flower.sh /celery_flower.sh
 COPY ./bin/dump_configuration.sh /dump_configuration.sh
+COPY ./bin/wait_for_db.sh /wait_for_db.sh
+COPY ./bin/setup_configuration.sh /setup_configuration.sh
```

Comment:
I don't know how relevant this PR still is, but I think the `SCRIPTPATH` [environment variable ](https://github.com/open-formulieren/open-forms/pull/4822/files#diff-052ea553068c1a5f498a2229cc723b3c7e7e5a73e5380b3bf52cf29f513cb82fR14)should be taken into account here

Reply:
I'm confused - where does that variable even get set? I did a search on the repo and nowhere is it defined.

Reply:
I couldn't find it either :sweat_smile:, because it probably always (because I didn't see any reference either) evaluates to an empty string everything keeps working :shrug:

Reply:
I think I was a bit too eager copying and pasting :sweat_smile: 

I'll remove it